src/Files.App/Utils/Storage/Helpers/FilePropertiesHelpers.cs

```diff
@@ -41,6 +41,10 @@ public static nint GetWindowHandle(Window w)
 		/// <param name="associatedInstance">Associated main window instance</param>
 		public static void OpenPropertiesWindow(IShellPage associatedInstance)
 		{
+			if (associatedInstance?.SlimContentPage is null ||
+				associatedInstance?.FilesystemViewModel is null)
+				return;
```

Comment:
This will probably work, but it is better to continue the process if possible to reduce the risk of regression.
Here is my suggestion:
```csharp
			if (associatedInstance is null)
				return;

			object item;

			var page = associatedInstance.SlimContentPage;

			// Item(s) selected
			if (page is not null && page.IsItemSelected)
			{
				// Selected item(s)
				item = page.SelectedItems?.Count is 1
					? page.SelectedItem!
					: page.SelectedItems!;
			}
			// No items selected
			else
			{
				// Instance's current folder
				var folder = associatedInstance.FilesystemViewModel?.CurrentFolder;
				if (folder is null)
					return;
```